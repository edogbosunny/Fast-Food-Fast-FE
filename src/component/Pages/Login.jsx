/* eslint-disable react/prefer-stateless-function */
import React, { Component } from 'react';
// import { connect } from 'react-redux';
import LoginCard from '../UI/cards/LoginCard.jsx';
import '../../styles/style.css';

class Login extends Component {
  render() {
    return (
      <div className='bg'>
        <LoginCard />
      </div>
    );
  }
}

export default Login;
