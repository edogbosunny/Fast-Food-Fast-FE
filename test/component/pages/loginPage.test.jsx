import React from 'react';
import { shallow } from 'enzyme';
import Login from '../../../src/component/Pages/Login';

describe('Login page', () => {
  const wrapper = shallow(
    <Login />,
  );
  it('should mount the login page', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
