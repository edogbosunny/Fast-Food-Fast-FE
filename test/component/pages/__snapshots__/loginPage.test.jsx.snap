// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login page should mount the login page 1`] = `
<div
  className="bg"
>
  <withRouter(Connect(LoginCard)) />
</div>
`;
